ruby/ql/lib/codeql/ruby/dataflow/internal/DataFlowDispatch.qll

```diff
@@ -464,9 +461,9 @@ private module Cached {
       )
     )
     or
-    exists(Module superClass, string method |
-      superCall(call, superClass, method) and
-      result = lookupMethod(superClass, method)
+    exists(Module cls, string method |
+      superCall(call, cls, method) and
+      result = lookupMethod(cls.getAnImmediateAncestor+(), method)
```

Comment:
Shouldn't this just be `getAnImmediateAncestor()` without the `+`? `lookupMethod` looks up in ancestors already.

Reply:
Good point!